ts/Series/Heatmap/HeatmapSeries.ts

```diff
@@ -608,42 +568,41 @@ class HeatmapSeries extends ScatterSeries {
                             Math.ceil(
                                 canvasWidth *
                                 yPlacement(
-                                    scaleToImg(
+                                    resizeFromRange(
                                         yMax - y,
                                         yMin,
-                                        yMax,
-                                        0,
+                                        yDelta,
                                         heightLastIndex
                                     )
                                 ) +
                                 xPlacement(
-                                    scaleToImg(
+                                    resizeFromRange(
                                         x,
                                         xMin,
-                                        xMax,
-                                        0,
+                                        xDelta,
                                         widthLastIndex
```

Comment:
If I change this from `widthLastIndex` to `canvasWidth`, the `studies/heatmap-interpolation` is correct. But then the other two demos break 🤷